mobile/src/main/java/org/hawkular/client/android/activity/LoginActivity.java

```diff
@@ -190,7 +191,7 @@ public void onSuccess(List<Persona> personas) {
             LoginActivity activity = (LoginActivity) getActivity();
 
             activity.setUpBackendCommunication(persona);
-            BackendClient.of(getActivity()).getEnvironments(new EnvironmentsCallback(persona));
+           // BackendClient.of(getActivity()).getEnvironments(new EnvironmentsCallback(persona));
         }
 
         @Override
```

Comment:
Yeah, it was crashing initially also. I tried fixing it in this PR. Doesn't work for you @m-murad ?